src/main/java/com/ethioclicks/entity/StudentInfo.java

```diff
@@ -0,0 +1,82 @@
+package com.ethioclicks.entity;
+
+import java.sql.Timestamp;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "studentinfo")
+public class StudentInfo {
+    @Column(name = "Id")
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+	private int id;
+    @Column(name = "FirstName")
+	private String firstName;
+    @Column(name = "LastName")
+	private String lastName;
+    @Column(name = "Gender")
+	private String gender;
+    @Column(name = "Department")
+	private String department;
```

Comment:
private Department department;
we need to create Department and Instructor class where 1 student have 1 deprtment, 1 department has head Instructor, 1 department has many students

Reply:
Added